flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/active/ActiveResourceManager.java

```diff
@@ -197,11 +197,12 @@ public void onPreviousAttemptWorkersRecovered(Collection<WorkerType> recoveredWo
 	}
 
 	@Override
-	public void onWorkerTerminated(ResourceID resourceId) {
-		log.info("Worker {} is terminated.", resourceId.getStringWithMetadata());
+	public void onWorkerTerminated(ResourceID resourceId, Exception cause) {
```

Comment:
If we don't really use `Exception cause` here, then I would suggest to simply change it into a `String diagnostics` parameter. We can later still change it if we need an `Exception` to be reported.